src/Components/fe-login/stories/index.stories.js

```diff
@@ -7,9 +7,7 @@ export default {
 };
 
 function Template() {
-  return html`
-    <fe-login @input-validation=${ev => console.log(ev.detail)}> </fe-login>
-  `;
+  return html` <fe-login> </fe-login> `;
```

Comment:
You need to check the login details event

Reply:
okay will check

Reply:
once completed, then make it resolve

Reply:
resolved and pushed the
code